submissions/ettalha0x/main.cpp

```diff
@@ -0,0 +1,131 @@
+/* ************************************************************************** */
```

Comment:
Please make sure only your solution is included in your PR.

Reply:
did you mean to remove the header ?